bin/vscout.dart

```diff
@@ -1,54 +1,73 @@
+                                           /@@@@@@@@@@@@@(                                         
+                                            .@@@@@@@@@@@.                                          
+                                             .@@@@@@@@@,                                           
+                                              .&@@@@@@                                             
+                                                %@@@&                                              
+                                                 %@/                                               
+                                                  .                                                                                                                                                                                          
+  """);
+  print('Welcome to Vscout');
+  print('Robotics scouting software');
+  print('For more information, visit\nhttps://vscout.readthedocs.io');
+  List<dynamic> runCommands = List();
   // Create a new database handler with empty constructor.
   DatabaseHandler databaseHandler = DatabaseHandler();
   // Run all constructor functions in async function to await database construction completion.
   //  This is to prevent calls to an unfinished database object.
   // Add database related commands only if database exists, else only add [init] and [config].
-  if (await databaseHandler.initializeDatabase()) {
-    commands.add(AddCommand());
-    commands.add(FindCommand());
-    commands.add(LsCommand());
-    commands.add(RmCommand());
-    commands.add(ShowCommand());
-    commands.add(UpdateCommand());
-  }
-  commands.add(InitCommand());
-  commands.add(ConfigCommand());
+
+  await databaseHandler.initializeDatabase();
+
   var runner = CommandRunner(
       'vscout',
       'Robotics scouting software'
           '\n\n'
           'https://vscout.readthedocs.io');
 
   runner.argParser.addFlag('verbose', negatable: false);
-  commands.forEach((command) => runner..addCommand(command));
-
-  return await runner.run(args).catchError((exception, stackTrace) {
-    if (exception is String) {
-      // STDOUT is buffered and writes are in batches.
-      stdout.writeln(exception);
-    } else {
-      // STDERR is unbuffered and every character is written as soon as it is available.
-      stderr.writeln('Error Message: $exception');
-      if (args.contains('--verbose')) {
-        stderr.writeln(stackTrace);
-      }
-    }
-    exitCode = generalError;
-  }).whenComplete(() {
-    // ANSI escape code to reset (all attributes off).
-    stdout.write(resetAll.wrap(''));
-  });
+  runner..addCommand(VscoutExecCommand());
+  runner..addCommand(VscoutCommand());
+  CliView view = CliView();
+  view.runner = runner;
+  Stream cmdLine = Utf8Decoder().bind(stdin).transform(InputArgsParser());
+  await view.listenTo(cmdLine);
+  await view.inputSubscription.asFuture();
+  print('Vscout concluded');
```

Comment:
"vscout exited with code 0" (ideally later), but for now "vscout exited"

Reply:
also notice my capitalization on vscout or the lack thereof (vscout) (Vscout only there for class names to comply with style guides)